src/network_event/client_event.rs

```diff
@@ -139,6 +139,7 @@ impl ClientEventAppExt for App {
         self.add_event::<T>()
             .init_resource::<Events<FromClient<T>>>()
             .insert_resource(ClientEventChannel::<T>::new(channel_id))
+            .add_systems(PreUpdate, reset_system::<T>.in_set(ClientSet::ResetEvents))
```

Comment:
Could you group all `PreUpdate` systems here and in `server_event`?
Just for consistency with Bevy and the rest of the codebase.